dolphinscheduler-api/pom.xml

```diff
@@ -64,6 +64,10 @@
             <groupId>org.apache.dolphinscheduler</groupId>
             <artifactId>dolphinscheduler-registry-zookeeper</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dolphinscheduler</groupId>
+            <artifactId>dolphinscheduler-registry-mysql</artifactId>
+        </dependency>
```

Comment:
Can you do in the same way as how we did in the task plugins? Create a new module `dolphinscheduler-registry-all` to contain all these registry plugin, and we can only add `dolphinscheduler-registry-all` to these starter module, in the future when we add new registry module we can only add them in `dolphinscheduler-registry-all` once

Reply:
Done